app/models/request.rb

```diff
@@ -1,4 +1,8 @@
 class Request < ApplicationRecord
   belongs_to :user
   belongs_to :expert
+  has_many :offers
+
+  validates :title, presence: true
+  validates :description, presence: true, length: { minimum: 25 }
 end
```

Comment:
should we add a validation of status? can only be certain value (pending, offer receveid, ... i think geoffroy has listed them on the trello or somewhere)